File: assignment2_q2.py
# Define vectors
vector_A = [4,2,6]
vector_B = [5,4,3]

print("A = ", vector_A)
print("B = ", vector_B)

# Size of vector
size = len(vector_A)

# Vector summation
sum = []
for i in range(size):
	sum.append(vector_A[i] + vector_B[i])

print("A+B = ", sum)

# Dot product of vectors
dot_product = 0
for i in range(size):
	pro = vector_A[i]*vector_B[i]
	dot_product += pro

print("A.B = ", dot_product)

File: assignment2_q2.c
#include<stdio.h>
int main(){

	int A[3]={4,2,6};
	int B[3]={5,4,3};
	int C[3] = {0, 0, 0};
	
	//A+B
	printf("A+B = ");
	for(int i=0; i<3; ++i){
		C[i] = A[i] + B[i];
		printf("'%d'", C[i]);
	}
	



	// AxB
	int product = 0;
	for(int i=0; i<3; ++i){
		product += A[i]*B[i];
	}
	printf("\nA.B = %d", product);

}